ansible/install_neutron-api.yaml

```diff
@@ -25,6 +29,32 @@
     - "ocp/neutron-api/2"
     - "ocp/neutron-api/3"
 
+  - name: get NBConnection
+    shell: >
+      oc get cm -n openstack ovn-connection -o json | jq '.data.NBConnection' -r
```

Comment:
Nit: if you add the -e argument to jq it will cause an error if the value doesn't exist. I've caught a couple of silent bugs this way.

Also 'set -e -o pipefail' for the same reason.

Same comment below, too.